dashboard/client/src/common/ProfilingLink.tsx

```diff
@@ -10,6 +10,49 @@ type CpuProfilingLinkProps = PropsWithChildren<
   } & ClassNameProps
 >;
 
+type TaskProfilingStackTraceProps = {
+  taskId: string | number | null | undefined;
```

Comment:
```suggestion
  taskId: string | null | undefined;
```

Reply:
Good catch, I double confirmed Task.task_id is a string